apps/docs/src/guide/provider/provider-options.md

```diff
@@ -39,3 +39,27 @@ Provide a custom `fetch` function that'll replace the default fetch call.
 _Note: If defined, `requestMiddleware`, `timeout` and `retryOptions` are applied to this custom `fetch` function as well._
 
 <<< @/../../docs-snippets/src/guide/provider/provider.test.ts#options-fetch{ts:line-numbers}
+
+### `cacheUtxo`
+
+When using the SDK, it may be necessary to submit multiple transactions from the same account in a short period. In such cases, the SDK creates and funds these transactions, then submits them to the node.
+
+However, if a second transaction is created before the first one is processed, there is a chance of using the same UTXO(s) for both transactions. This happens because the UTXO(s) used in the first transaction are still unspent until the transaction is fully processed.
+
```

Comment:
Might be beneficial to include a link above the [UTXO architecture](https://docs.fuel.network/docs/fuels-ts/wallets/checking-balances/#understanding-the-utxo-model) docs.

Reply:
Unrelated question outside of the scope of this PR, but should this [UTXO architecture](https://docs.fuel.network/docs/fuels-ts/wallets/checking-balances/#understanding-the-utxo-model) be its own thing?

I mean, moved outside of `Wallet` > `Checking Balances`.

Reply:
@arboleya https://github.com/FuelLabs/fuels-ts/pull/2624